### PR TITLE
Add MongoDB replicaset and Elasticsearch optional dependency charts

### PR DESCRIPTION
**Pull request by previ**
This PR adds optional MongoDB replicaset and Elasticsearch as dependency charts.

It also adds MongoDB authentication (optional).

### CODE REVIEW

**Comment by previ** [comment]
changes committed

File: gravitee/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: elasticsearch
+  version: ^1.22.0
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  condition: es.enabled
```
Comment (brasseld) [inline gravitee/requirements.yaml]: @previ Please update the README to explain how to disable / enable ES dependency while installing the chart (`--set es.enabled=false`) and the default value.

File: gravitee/requirements.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- name: mongodb-replicaset
+  version: ^3.9.0
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  condition: mongodb.enabled
```
Comment (brasseld) [inline gravitee/requirements.yaml]: @previ Please update the README to explain how to disable / enable MongoDB dependency while installing the chart (`--set es.enabled=false`) and the default value.

File: gravitee/values.yaml [hunk 3]
```diff
@@ -29,6 +29,42 @@ mongo:
...
+
+mongodb-replicaset:
+  enabled: true
+  replicas: 3
+  replicaSetName: rs0
+  auth:
+    enabled: false
+    adminUser: username
+    adminPassword: password
+    metricsUser: metrics
```
Comment (brasseld) [inline gravitee/values.yaml]: What is this metricsUser ?

File: gravitee/Chart.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Gravitee Helm chart for Kubernetes
 name: gravitee
-version: 1.23.1
+version: 1.23.2
```
Comment (brasseld) [inline gravitee/Chart.yaml]: Shouldn't be 1.26.1 to align the gravitee version?

File: gravitee/README.md [hunk 5]
```diff
@@ -24,10 +25,9 @@ $ helm package .
 To install the chart using the chart archive, run:
 
 ```
-$ helm install gravitee-1.23.1.tgz
+$ helm install gravitee-1.23.2.tgz
```
Comment (brasseld) [inline gravitee/README.md]: Shouldn't be 1.26.1 to align the gravitee version?

File: gravitee/README.md [hunk 6]
```diff
@@ -1,15 +1,16 @@
...
 
 ## Chart Details
 
 This chart will deploy the following:
 
-
 - Gravitee UI
 - Gravitee API
 - Gravitee Gateway
+- MongoDB replicaset (optionally)
```
Comment (brasseld) [inline gravitee/README.md]: Define optionally : how to enable / disable

File: gravitee/values.yaml (hunk 3 above)
Comment (previ) [reply gravitee/values.yaml]: it's for the mongodb exporter (for prometheus) (if enabled).

**Comment by brasseld** [comment]
Sounds good @previ 

Thanks a lot !

**Pull request merged**